server/src/main/java/com/octopus/teamcity/opentelemetry/server/TeamCityBuildListener.java

```diff
@@ -263,13 +267,17 @@ public boolean acceptMessage(LogMessage message, boolean lastMessageInParent) {
         return buildLogs;
     }
 
-    private void setArtifactAttributes(SRunningBuild build, String buildName, Span span) {
-        Loggers.SERVER.info("OTEL_PLUGIN: Retrieving build artifact attributes for build " + buildName);
-        BuildArtifacts buildArtifacts = build.getArtifacts(BuildArtifactsViewMode.VIEW_ALL);
+    private void setArtifactAttributes(SRunningBuild build, Span span, String spanId) {
+        Loggers.SERVER.info("OTEL_PLUGIN: Retrieving build artifact attributes for build " + getBuildName(build));
```

Comment:
Should this be debug?
```suggestion
        Loggers.SERVER.debug("OTEL_PLUGIN: Retrieving build artifact attributes for build " + getBuildName(build));
```

Reply:
do we want to add build id here?